apps/language_server/lib/language_server/experimental/provider/handlers/goto_definition.ex

```diff
@@ -0,0 +1,27 @@
+defmodule ElixirLS.LanguageServer.Experimental.Provider.Handlers.GotoDefinition do
+  alias ElixirLS.LanguageServer.Experimental.Protocol.Requests.GotoDefinition
+  alias ElixirLS.LanguageServer.Experimental.Protocol.Responses
+  alias ElixirLS.LanguageServer.Experimental.CodeMod.Location, as: CodeModLocation
+  alias ElixirLS.LanguageServer.Experimental.SourceFile
+  require Logger
+
+  def handle(%GotoDefinition{} = request, _) do
+    source_file = request.source_file
+    pos = request.position
+
+    source_file_string = source_file |> SourceFile.to_string()
```

Comment:
why use a pipeline here? It's just a straight function call
```suggestion
    source_file_string = SourceFile.to_string(source_file)
```